test/e2e/antctl_test.go

```diff
@@ -59,6 +60,24 @@ func TestAntctl(t *testing.T) {
 	})
 }
 
+// TestAntctlFlowAggregator is the top-level test which contains the
+// Flow Aggregator Antctl related test cases
+func TestAntctlFlowAggregator(t *testing.T) {
+	skipIfHasWindowsNodes(t)
+	skipIfNotRequired(t, "mode-irrelevant")
+
+	data, _, _, err := setupTestWithIPFIXCollector(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownFlowAggregator(t, data)
+	defer teardownTest(t, data)
+
+	t.Run("testAntctlFlowAggregatorLocalAccess", func(t *testing.T) {
+		testAntctlFlowAggregatorLocalAccess(t, data)
+	})
+}
```

Comment:
I think it is unfortunate that we have to setup everything just for this small test, then tear it down. Can we merge this test with the other FlowAggregator e2e tests instead, so we only do the set up once?

Reply:
I have tried to merge the tests, but noticed some of the `testAntctlControllerRemoteAccess` may fail when using coverage on CI. Still looking for the reason.

Reply:
solved

Reply:
it's not really what I had in mind. We are still calling `setupTestWithIPFIXCollector` twice (once for antctl tests and once for the Flow Aggregator tests). We should try to call it once IMO, and move the antctl tests for the Flow Aggregator to `test/e2e/flowaggregator_test.go`.

Reply:
Got it, thanks. Updated.